bin/game.rb

```diff
@@ -0,0 +1,48 @@
+# Class Game is the main class
+class Game
+  require '../lib/board'
+  require '../lib/message'
+
+  def initialize
+    @turn = 1
+    @winner = false
+    @game_over = false
+
+    @board = Board.new
+    move
+  end
+
+  def move
+    @board.show_board
+    if @game_over
+      if @winner
+        Message.new(:winner, @turn)
+      else
+        Message.new(:tie)
+      end
+    else
+      Message.new(:instruction)
+      Message.new(:turn, @turn)
+      field = gets.chomp
+      play(field)
+    end
+  end
```

Comment:
This method does three things:
- Print the board
- Check if the game is over and if there is a winner
- Get user input.

Can you split into different methods and then use those methods in another method that contains the main structure/flow of the game?